repos.yaml

```diff
@@ -10,3 +10,10 @@
 #      - name: foo
 #        email: foo@example.com
 #
+
+repositories:
+  - name: kubernetes/ingress-nginx
```

Comment:
Namespaces cannot contain slashes

Reply:
it should look as `kubernetes-ingress-nginx`

Reply:
fixed